resources/object_repo/import_dbaasinventory_obj.resource

```diff
@@ -27,8 +27,9 @@ ${btnCreateEnabled}             //button[contains(.,"Create") and @aria-disabled
 ${lblDBInstSuccess}             //h2[contains(.,"Database instances") and contains(.,"successful")]
 ${btnViewProviderAcc}           //button[contains(.,"View Provider Accounts")]
 ${tblInstanceTable}             id:instance-table
-${btnMenuDropDown}              //button[contains(.,"<<menu>>")]
-${btnMenuOptDropDown}           //a[contains(.,"<<menuopt>>")]
+${btnMenuDropDown}              //*[@id="toggle-id-4"]
```

Comment:
These might removed based on the above comments

Reply:
Look at my previous comments. The "<<menu>>" does not exist anymore in the DOM and it fails all the tests.

Reply:
OK